### ISSUE TITLE
Evidence level - swap 'C' and 'D' for all evidence items - server side

### ISSUE THREAD

**Issue by malachig** [genome/civic-server#138]
After feedback from clinicians and users we have decided that the order of evidence levels should be adjusted to:

- A	Validated
- B	Clinical
- C	Case Study
- D	Preclinical
- E	Inferential

So `C` and `D` should be swapped. The reasoning here is that, even though the case study only involves a single individual, at least it is a real human patient.  Pre-clinical is often based on mouse models, cell lines, etc.

Implementation will require changes to both server and client.
- In the server it should be as simple as changing all `C` to `D` and all `D` to `C`.
- In the client we will just need to change various labels and help docs. We will create a separate issue for that.

**Comment by kkrysiak** [comment]
We will need to also update the glossary server side.

**Comment by kkrysiak** [comment]
Should we include a page with a disclaimer for all of the previous monthly/nightly data dumps prior to this change informing users of this change?

**Comment by malachig** [comment]
@jmcmichael has made this change on the client side.  We need to couple this closely to the change on the server side or a great deal of confusion could result.

**Comment by acoffman** [comment]
@malachig Yup! @jmcmichael and I will coordinate a release where we will do the following:

 * Change the labels server-side
 * Push a new version with both the client and server side change
 * Swap the existing `C` and `D` values in the database

So as to minimize any potential window of confusion.

**Comment by kkrysiak** [comment]
We will also need to update the help pages accordingly.

**Comment by acoffman** [comment]
Complete and pushed. Waiting on puppet to deploy to prod.
